langchain/src/util/convert-to-openai-function.ts

```diff
@@ -0,0 +1,485 @@
+import 'reflect-metadata'
```

Comment:
This looks it's going to be legacy soon with this new proposal?

https://www.npmjs.com/package/reflect-metadata

How does this look for end users? Are the decorators compiled to JS somehow from TypeScript?

I haven't worked with them much in JS

Reply:
For the users, it would look exactly how I have it written in the examples